src/styles/Button.css

```diff
@@ -0,0 +1,5 @@
+.button {
+  display: flex;
+  justify-content: center;
+  margin-top: 5%;
```

Comment:
Would it better to define margins in pixels or in percentage? I'm asking because in design we didn't take that into account much 😅

Reply:
If we are going to do a responsive app, percentage is better. 
But, we should not define it as constants but as bootstrap/css using their keywords.
https://getbootstrap.com/docs/3.4/css/
For tailwind which I prefer https://tailwindcss.com/docs/responsive-design